docker/incremental_load.sh.tpl

```diff
@@ -39,8 +49,8 @@ IMAGE_LEN=$(for i in $IMAGES; do echo -n $i | wc -c; done | sort -g | head -1 |
 [ -n "$IMAGE_LEN" ] || IMAGE_LEN=64
 
 # Create temporary files in which to record things to clean up.
-TEMP_FILES="$(mktemp -t)"
-TEMP_IMAGES="$(mktemp -t)"
+TEMP_FILES="$(mktemp -t 2>/dev/null || mktemp -t 'rules_docker_files')"
```

Comment:
Will be a bit of a delay before I can look at that, should be able to get to it Fri or Sat.  Either hold the merge until then or I'll send in another PR.